wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandControllerPOV.java

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj2.command;
+
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj2.command.button.POVButton;
+
+/**
+ * Provides POVButtons for binding commands to any GenericHID inherited class.
+ */
+public class CommandControllerPOV {
+  private final GenericHID m_hid;
+  private final int m_povNumber;
+
+  private POVButton m_upButton; // 0 degrees
+  private POVButton m_upRightButton; // 45 degrees
+  private POVButton m_rightButton; // 90 degrees
+  private POVButton m_downRightButton; // 135 degrees
+  private POVButton m_downButton; // 180 degrees
+  private POVButton m_downLeftButton; // 225 degrees
+  private POVButton m_leftButton; // 270 degrees
+  private POVButton m_upLeftButton; // 315 degrees
+
+  /**
+   * Constructs a ControllerPOV.
+   * 
+   * @param hid The HID controller to read the POV from.
+   */
+  public CommandControllerPOV(GenericHID hid) {
+    m_hid = hid;
+    m_povNumber = 0;
+  }
+
+  /**
+   * Constructs a ControllerPOV.
+   * 
+   * @param hid The HID controller to read the POV from.
+   * @param povNumber The controller POV index to use.
+   */
+  public CommandControllerPOV(GenericHID hid, int povNumber) {
+    m_hid = hid;
+    m_povNumber = povNumber;
+  }
+
+  /**
+   * Returns the upper (0 degrees) POVButton object.
+   *
+   * <p>To get its value, use {@link POVButton#get()}.
+   */
+  @SuppressWarnings("checkstyle:MethodName")
+  public POVButton uButton() {
```

Comment:
I thought this was getting renamed to `up()`.

Reply:
Looks like I missed that one.